src/auth_api/orchestrator.py

```diff
@@ -119,7 +127,7 @@ def _get_next_step(
         self
     ) -> NextStep:
         """
-        Controls the flow of the onboarding
+        Flow control of the onboarding.
```

Comment:
```suggestion
        Flow control of the onboarding.

```
Add new line between summary and extended summary.